### PR TITLE
chore(Dockerfile): update Workflow CLI to latest published

### PR DESCRIPTION
**Pull request by mboersma**
Closes #216.

### CODE REVIEW

**Comment by mboersma** [comment]
I also added a commit that marks one Ginkgo test spec as pending, because it seems to fail often with the current `deis` CLI (for me locally as well).

File: tests/apps_test.go [hunk 1]
```diff
@@ -204,7 +204,8 @@ var _ = Describe("deis apps", func() {
 				Eventually(sess).Should(Exit(0))
 			})
 
-			Specify("that user can run a command with quotes in that app's environment", func() {
+			// FIXME: why is Workflow failing this test?
```
Comment (arschles) [inline tests/apps_test.go]: is there an issue for this?

**Comment by arschles** [comment]
LGTM3, although an issue referenced in https://github.com/deis/workflow-e2e/pull/217#discussion_r65396318 would be helpful

File: tests/apps_test.go (hunk 1 above)
Comment (mboersma) [inline tests/apps_test.go]: Assuming this PR passes--that the `FIXME:` is needed--I'll create an issue to track it before I merge this.

**Pull request merged**